content/articles/2018-11-tibble-2.0.0-announce.Rmd

```diff
@@ -0,0 +1,158 @@
+---
+title: Announcing tibble 2.0.0
+slug: tibble-2.0.0-announce
+description: >
+    Tibbles are a modern reimagining of the data frame, keeping what time has shown to be effective, and throwing out what is not, with nicer default output too! The upcoming 2.0.0 release adds a slew of features but also changes some internals, which in some cases might require adaptation. This blog post describes some of the stumbling blocks we've encountered when trying to adapt other packages to the dev version of tibble.
+date: 2018-11-21
+author: Kirill Müller, Jenny Bryan
+photo:
+  url: https://unsplash.com/photos/yYawh30qf28
+  author: Gabriel Porras
+categories: [package]
+---
+
+
+```{r setup, include = FALSE}
+library(tibble)
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+Version 2.0.0 of the _tibble_ package is almost ready for release.
+Tibbles are a modern reimagining of the data frame, keeping what time has shown to be effective, and throwing out what is not, with nicer default output too! Grab the development version with:
+
+```r
+remotes::install_github("tidyverse/tibble")
+```
+
+We announce this release before actually submitting to CRAN, because some changes require attention from authors of packages that import or otherwise depend on _tibble_.
+We would like to use this opportunity to gather feedback and perhaps rollback or soften some of the most disruptive changes.
+This blog post is mostly interesting to package developers, new features and a higher-level overview will be presented in a separate blog post.
+
+We tested ~3000 CRAN and Bioconductor packages that depend directly or indirectly on _tibble_ with `R CMD check`, and compared check results for the CRAN version with our results.
+We notified the maintainers of the packages, and aim for a CRAN release before Christmas, just in time for [rstudio::conf](https://www.rstudio.com/conference/).
+
+In the following we showcase the changes that affected many of the now failing packages that we analyzed, and suggest solutions.
+For the full list of changes, features and bug fixes, please see the [release notes](https://github.com/tidyverse/tibble/tree/master/NEWS.md) for a complete list.
+
+
+## Construction and validation
+
+The `new_tibble()` function now more closely follow the advice in [adv-r](https://adv-r.hadley.nz/s3.html#s3-classes).
+Specifically:
+
+- `new_tibble()` is very fast again and does very little checking itself,
+- The new `validate_tibble()` is responsible for validating the structure of a tibble.
+
+This means that the `nrow` argument to `new_tibble()` is now mandatory.
```

Comment:
Emphasise that you should use `new_tibble()` if you want performance, and you're very confident that the inputs are correct. Most people should use `tibble()`